src/main/java/com/example/WMS/thread/ReadThread.java

```diff
@@ -0,0 +1,21 @@
+package com.example.WMS.thread;
+
+import com.example.WMS.entity.Order;
+import com.example.WMS.entity.ProductDetails;
+import com.example.WMS.handlers.ProductDetailsHandler;
+import com.example.WMS.handlers.ReadWriteHandler;
+
+public class ReadThread extends Thread {
```

Comment:
If this class is not used, please remove

Reply:
removed 2 unwanted classes!